3604/1/table_output.rb

```diff
@@ -0,0 +1,41 @@
+require 'terminal-table'
+
+# output table with all information about gems
+class TableOutput
+  HASH_FOR_TABLE = {
+    name: '',
+    used_by: 'used by ',
+    watched_by: 'watched by ',
+    stars: 'stars ',
+    forks: 'forks ',
+    contributors: 'contributors ',
+    issues: 'issues '
+  }.freeze
+
+  def table_output(array_gems, count_gems_in_argv)
+    table = Terminal::Table.new
+    table.style = { border_top: false, border_bottom: false }
+    how_many_names_need_to_display(count_gems_in_argv, array_gems.size).times do |iter|
```

Comment:
`count_gems_in_argv.to_i` and then remove `.to_i` occurrences in `how_many_names_need_to_display`